debian/patches/cpick-d75840be-fix-retry-AWS-hotplug-for-async-IMDS-5995

```diff
@@ -0,0 +1,247 @@
+From d75840bee572d097be2403e971da71ad1776f47f Mon Sep 17 00:00:00 2001
+From: Brett Holman <brett.holman@canonical.com>
+Date: Mon, 3 Feb 2025 10:20:46 -0700
```

Comment:
In the past, Andreas requested we include `Bug` (for upstream) and/or `Bug-Ubuntu` (for downstream) references when applicable. They're part of the dep-3 standard, and I think a Github reference would be helpful here.

Reply:
And actually, I think we need the downstream bug too. This hotfix branch isn't going through our normal SRU exception process and instead is highlighting individual bugs for each change, so I think we need one here too.

Reply:
Yeah I knew we needed a launchpad bug for the release, I just didn't realize that it needed to be in the patch.